pages/faq/troubleshooting/troubleshooting.md

```diff
@@ -14,6 +14,8 @@ title: Troubleshooting
   * [Connectivity](#connectivity)
   * [SD Card Corruption](#sd-card-corruption)
   * [Wifi doesn't connect on RPI3](#wifi-connection-problems-on-rpi3)
+* [Balena FIN](#balena-fin)
```

Comment:
According to the [Style Guide](https://github.com/balena-io/docs#balena-specific-words), for consistency, we try to mention `balenaFin` in this way only. Please replace balenaFin across your PR to match the style guide